README.md

```diff
@@ -34,16 +36,41 @@ tenant, err := tenant.Create(&tenant.TenantParams{})
 Instantiate the Warrant client with your API key to get started:
 ```go
 import "github.com/warrant-dev/warrant-go"
+import "github.com/warrant-dev/warrant-go/v3/config"
 
 client := warrant.NewClient(config.ClientConfig{
     ApiKey: "api_test_f5dsKVeYnVSLHGje44zAygqgqXiLJBICbFzCiAg1E=",
 	ApiEndpoint: "https://api.warrant.dev",
 	AuthorizeEndpoint: "https://api.warrant.dev",
-	SelfServiceDashEndpoint: "https://self-serve.warrant.dev"
+	SelfServiceDashEndpoint: "https://self-serve.warrant.dev",
+})
+```
+
+## Configuring Endpoints
+---
+The API, Authorize, and Self-Service endpoints the SDK makes requests to is configurable via the `warrant.ApiEndpoint`, `warrant.AuthorizeEndpoint`, `warrant.SelfServiceDashEndpoint` attributes:
```

Comment:
The API, Authorize, and Self-Service endpoints the SDK makes requests to ~is~ are configurable via the `warrant.ApiEndpoint`, `warrant.AuthorizeEndpoint`, `warrant.SelfServiceDashEndpoint` attributes: